query.go

```diff
@@ -646,18 +769,13 @@ func (s *ChainService) prepareCFiltersQuery(blockHash chainhash.Hash,
 		startHeight = int64(height)
 		stopHeight = startHeight + batchSize - 1
 
-		// We need a longer timeout, since we are going to receive more
-		// than a single response.
-		options = append(options, Timeout(QueryBatchTimeout))
-
 	// Reverse batch, fetch as many of the preceding filters as possible.
 	case reverseBatch:
 		stopHeight = int64(height)
 		startHeight = stopHeight - batchSize + 1
 
-		// We need a longer timeout, since we are going to receive more
-		// than a single response.
-		options = append(options, Timeout(QueryBatchTimeout))
```

Comment:
Why do we no longer need the batch timeout?

Reply:
imo the dynamic timeout is way nicer cause then we dont wait for 30 seconds on a non-responsive peer when all we want is 2 filters.